css/style.css

```diff
@@ -16,4 +16,243 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+ /* Header */
 
+ header {
+     display: flex;
+     justify-content: space-between;
+     align-items: center;
+     padding: 1rem 3rem;
+ }
+
+ .svg-logo {
+     height: 2.5rem;
+ }
+
+ @media only screen and (max-width: 768px) {
+
+     header {
+     padding: 1rem 1.2rem;
+    }
+  
+   .svg-logo {
+     height: 2rem;
+    }
+}
+
+/* navbar*/
+
+ .nav-bar {
+     width: 50%;
+ }
+
+.navigation__list {
+  display: flex;
```

Comment:
hurrah for well implemented flex boxes 🍾

Reply:
Thanks again. I wanted to be comfortable with the flexbox.